app/src/index.js

```diff
@@ -19,6 +21,10 @@ class App extends React.Component {
       <div className='app'>
         { target !== 'cumulus' ? (
           <div className='app__target--container'>
+            { /*  Need to add alert function
```

Comment:
What's happening here? Does this functionality need to go into this PR?

Reply:
Since I was working modals, I was looking at pop-ups as well and just was making notes as I went along so I don't forget to add that. It can be removed if you want for now. I have it saved locally.